### PR TITLE
feat: add supabase completion spec

### PR DESCRIPTION
**Pull request by lyqht**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**

Closes #713, closes https://github.com/withfig/autocomplete/issues/1556

This spec is autogenerated using cobra.
Thanks @fedeci for pointing out the guide on Fig! 

P.S. please add hacktoberfest label 🐱 

**Screenshots**

Fig-SupabaseCLI-Demo

### CODE REVIEW

**Comment by fedeci** [comment]
Ref #713

File: src/supabase.ts [hunk 1]
```diff
@@ -0,0 +1,483 @@
...
+          ],
+        },
+        {
+          name: "diff",
+          description: "Diffs the local database for schema changes",
+          options: [
+            {
+              name: ["--file", "-f"],
+              description: "Saves schema diff to a file",
+              args: { name: "file" },
```
Comment (fedeci) [inline src/supabase.ts:72]:
```suggestion
              args: {
                name: "file",
                template: "filepaths",
                suggestCurrentToken: true,
              },
```

File: src/supabase.ts [hunk 2]
```diff
@@ -0,0 +1,483 @@
...
+        { name: "new", description: "Create a new Function locally" },
+        {
+          name: "serve",
+          description: "Serve a Function locally",
+          options: [
+            {
+              name: "--env-file",
+              description:
+                "Path to an env file to be populated to the Function environment",
+              args: { name: "env-file" },
```
Comment (fedeci) [inline src/supabase.ts:186]:
```suggestion
              args: {
                name: "env-file",
                template: "filepaths",
              },
```

File: src/supabase.ts [hunk 3]
```diff
@@ -0,0 +1,483 @@
...
+      subcommands: [
+        { name: "list", description: "List all secrets on Supabase" },
+        {
+          name: "set",
+          description: "Set a secret(s) on Supabase",
+          options: [
+            {
+              name: "--env-file",
+              description: "Read secrets from a .env file",
+              args: { name: "env-file" },
```
Comment (fedeci) [inline src/supabase.ts:315]:
```suggestion
              args: {
                name: "env-file",
                template: "filepaths",
              },
```

**Review by fedeci — CHANGES_REQUESTED** [review]
Thanks! Just a couple issues to fix, it would be cool to have generators in the future!

**Comment by lyqht** [comment]
Thanks @fedeci, have committed your suggestions.

**Review by fedeci — APPROVED** [review]

**Comment by fedeci** [comment]
Thanks!

**Pull request merged**